app/src/main/java/org/sopt/official/feature/auth/component/AuthButton.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * MIT License
+ * Copyright 2024 SOPT - Shout Our Passion Together
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.sopt.official.feature.auth.component
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.unit.dp
+import org.sopt.official.designsystem.Black
+import org.sopt.official.designsystem.Black80
+import org.sopt.official.designsystem.Gray60
+import org.sopt.official.designsystem.White
+
+@Composable
+internal fun AuthButton(
+    padding: PaddingValues,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    shape: Shape = RoundedCornerShape(10.dp),
+    containerColor: Color = White,
+    contentColor: Color = Black,
+    isEnabled: Boolean = true,
+    disabledContainerColor: Color = Black80,
+    disabledContentColor: Color = Gray60,
```

Comment:
```suggestion
    isEnabled: Boolean = true,
    containerColor: Color = White,
    contentColor: Color = Black,
    disabledContainerColor: Color = Black80,
    disabledContentColor: Color = Gray60,
```

~Defaults object를 활용해서 컬러 시스템을 하나의 객체 안에서 관리할 수 있게하는 패턴이 있습니당!

Reply:
https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/material/material/src/commonMain/kotlin/androidx/compose/material/TextFieldDefaults.kt?q=TextFieldDefaults

나중에 이거 참고해서 리팩토링해도 괜찮을 듯 (우선순위 하~최하)